fluent-bundle/src/resolver.ts

```diff
@@ -156,14 +156,14 @@ function resolveVariableReference(
   let arg: FluentVariable;
   if (scope.params) {
     // We're inside a TermReference. It's OK to reference undefined parameters.
-    if (Object.prototype.hasOwnProperty.call(scope.params, name)) {
+    if (Reflect.has(scope.params, name)) {
```

Comment:
`Reflect.has` is not available in IE11. This project doesn't provide details about the level of support that it is offering for legacy browsers, judging by the code, I don't see any usage of `Reflect`. We can replace this with `name in scope.params` which is equivalent and does work in legacy systems as well.